products/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+from .views import ListView
+
+urlpatterns = [
+    path("/list", ListView.as_view()),
```

Comment:
```suggestion
    path("", ListView.as_view()),
```
RESTful API 세션 때 들으셨겠지만, /products 로도 제품들 정보 충분히 나타낼 수 있습니다.

Reply:
넵! 충분히 인지하겠습니다!